### ISSUE TITLE
paper-tabs focus state issue

### ISSUE THREAD

**Issue by lpalmaro** [PolymerElements/paper-tabs#8]
The focus state is not visible. We need a clearer way to identify which item is in focus.

**Comment by cdata** [comment]
This has been fixed. Please re-open if it needs more work!

**Comment by myfreeweb** [comment]
Bold text is not very visible! Why not just add `paper-inky-focus-behavior` to `paper-tab`?

**Comment by bicknellr** [comment]
Thanks for the suggestion, @myfreeweb! I have to say I'm surprised we haven't done that already ... but everything with a sufficient combination of useful + simple always seems obvious after its been introduced. :) `paper-tab` already imports everything that PaperInkyFocusBehavior does anyway and getting rid of the bold text would be great too - it causes the tabs to resize slightly on focus and look janky. `paper-ripple` and `paper-inky-focus-behavior` might need to be tweaked a little. I'll open a new issue to track this.

**Comment by myfreeweb** [comment]
Yeah, you'll need to tweak it to make sure clicking on a tab doesn't cause two ripples (on the clicked tab + on the previously focused tab).

**Comment by bicknellr** [comment]
Opened as #150.
